config/plugin.properties

```diff
@@ -0,0 +1,92 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# This mapping is used to resolve the Jar package name without version (or call artifactId)
+#
+# corresponding to the module in the user Config, helping SeaTunnel to load the correct Jar package.
+# Don't modify the delimiter " -- ", just select the plugin you need
+--flink-sql-connectors--
```

Comment:
This file is not `.properties` format but use a `.properties` file extension, this is not a good ideal it can confuse users.

Reply:
hi, thanks for your review, this is our custom format, maybe it would be better not to add a suffix, I modified it, you can take a look